### ISSUE TITLE
Optimal parameters

### ISSUE THREAD

**Issue by dfx1822375** [timeseriesAI/tsai#538]
hi：@oguiza
    I would like to know how to save a network as an optimal network in tsai. For example, when I use `learn.fit_one_cycle(100,` `lr),` the accuracy of the network is changing. When the training is over, will the network save the epoch with the highest accuracy by default? If not, how do I make the network change to the epoch with the highest precision?
    Also, how to add the accuracy of the test set to the output of each epoch in tsai, I only found the loss in training and the accuracy of the validation set.

**Comment by zmce2018** [comment]
Hi @dfx1822375, 

You find the solution here https://timeseriesai.github.io/tsai/callback.core.html#SaveModel. 

You can include the SaveModel in the cbs in your function. Let me know if you need more help.
